.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: script
+
+env:
+  global:
+     CF_TARGET_URL="https://api.ng.bluemix.net"
+
+services:
+
+script:
+- bash -n $(find . -name *.sh)
```

Comment:
This appears to check for any file with a .sh extension before opening a bash terminal. Isn't our intent to check for verify-links.sh?

I'm not convinced this line is required - i think we can just run the .sh script in the next line.

Reply:
the "-n" flag on there just does a syntax check on the sh files in the repo. Not a big check, but at least it'll catch obvious things.

Reply:
okay, sgtm